src/Components/MenuItems/index.jsx

```diff
@@ -0,0 +1,33 @@
+import ItemInfo from "../ItemInfo";
+
+function MenuItems({ foodName, foodType, calories, price, side }) {
```

Comment:
This is actually a menu item. Not plural.

Reply:
Done